webextensions/api/scripting.json

```diff
@@ -67,8 +67,7 @@
                 },
                 "edge": "mirror",
                 "firefox": {
-                  "version_added": false,
-                  "impl_url": "https://bugzil.la/1736575"
```

Comment:
Also drop the note from https://github.com/mdn/browser-compat-data/blob/6bba19cc5fed4090e2e4fe67eec5a3d4d7994307/webextensions/api/scripting.json#L312